src/extension.ts

```diff
@@ -1097,6 +1098,24 @@ async function getLogsForPod(pod: PodSummary, follow?: boolean) {
     }
 }
 
+async function getEvents(follow: kuberesources.EventDisplayMode, explorerNode?: explorer.ResourceNode) {
+    let currentNS;
```

Comment:
rename this var, since it's not always "current" if the explorerNode is provided.